### PR TITLE
LPD-37699 Add new test to cover not instanceable widgets case

### PR DESCRIPTION
**Pull request by sandrodw3**

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by sandrodw3** [comment]
ci:test:page-management

**Comment by sandrodw3** [comment]
ci:test:page-management-playwright

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/WidgetsManager.js [hunk 1]
```diff
@@ -37,6 +37,14 @@ export default function WidgetsManager() {
 		selectWidgetFragmentEntryLinks
 	);
 
+	useEffect(() => {
+		dispatch(
+			loadWidgets({
+				fragmentEntryLinks: fragmentEntryLinksRef.current,
+			})
+		);
+	}, [dispatch, fragmentEntryLinksRef]);
```
Comment (victorg1991) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/WidgetsManager.js:46]: We can probably avoid this checking if there is a collection display, right?

Comment (sandrodw3) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/WidgetsManager.js:46]: Um, good idea :) I can then leave all the old logic and just put a `useEffect` in `WidgetsManager` to load them if they are not loaded and there's a collection display 🤔

Comment (victorg1991) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/WidgetsManager.js:46]: Make sense, this way we can still benefit for not loading the widget every single time :D

**Comment by ealonso** [comment]
ci:test:page-management:dryrun

**Comment by sandrodw3** [comment]
ci:test:page-management

**Comment by sandrodw3** [comment]
ci:test:page-management

**Comment by sandrodw3** [comment]
ci:test:page-management

**Comment by sandrodw3** [comment]
ci:test:page-management

**Comment by sandrodw3** [comment]
ci:test:page-management

**Comment by ealonso** [comment]
@sandrodw3

**Comment by ealonso** [comment]
ci:forward

**Comment by sandrodw3** [comment]
ci:stop

**Comment by sandrodw3** [comment]
https://github.com/brianchandotcom/liferay-portal/pull/155221

**Pull request closed without merging**